package.json

```diff
@@ -99,5 +99,8 @@
     "zx": "6.1.0"
   },
   "packageManager": "yarn@3.2.1",
-  "npmClient": "yarn"
+  "npmClient": "yarn",
+  "dependencies": {
```

Comment:
I'm not sure we should have this entire package as a dependency. Is it only used in the tests?